mParticle-Apple-SDK/Utils/NSDictionary+MPCaseInsensitive.m

```diff
@@ -72,8 +72,7 @@ - (id)valueForCaseInsensitiveKey:(NSString *)key {
             transformedDictionary[key] = [[NSString alloc] initWithData:obj encoding:NSUTF8StringEncoding];
         } else {
             MPILogError(@"Data type is not supported as an attribute value: %@ - %@", obj, [[obj class] description]);
```

Comment:
Question for reviewers: Should I keep this warning in?

Reply:
Seems like we should add another if else case for dictionaries?